multicluster/test/yamls/clusterset.yml

```diff
@@ -1,17 +1,15 @@
-apiVersion: multicluster.crd.antrea.io/v1alpha1
+apiVersion: multicluster.crd.antrea.io/v1alpha2
 kind: ClusterClaim
 metadata:
-  name: leadercluster-id
-  namespace: antrea-multicluster
-name: id.k8s.io
+  name: id.k8s.io
+  namespace: antrea-mcs-ns
```

Comment:
Please make sure you change it correctly when you are trying to resolve conflicts.
Namespace should be `antrea-multicluster`.